src/client/pages/AssessmentSection/Descriptions/Descriptions.tsx

```diff
@@ -8,10 +7,10 @@ import { useHasOriginalDataPointData } from '@client/store/pages/assessmentSecti
 import { useIsPrint } from '@client/hooks/useIsPath'
 
 import AnalysisDescriptions from './AnalysisDescriptions'
-import CommentableDescription from './CommentableDescription'
 import NationalDataDescriptions from './NationalDataDescriptions'
 
 type Props = {
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
Do we need this eslint disable ?

Reply:
In this case for some weird reason, it is throwing this error:
`ESLint: 'descriptions' PropType is defined but prop is never used(react/no-unused-prop-types)`
even though we are using it in sub class

Reply:
then desctructure props and pass each individual prop where needed